Cargo.toml

```diff
@@ -45,7 +48,7 @@ sp-core = { git = "https://github.com/paritytech/polkadot-sdk.git", tag = "polka
 sp-io = { git = "https://github.com/paritytech/polkadot-sdk.git", tag = "polkadot-v1.9.0", default-features = false }
 sp-runtime = { git = "https://github.com/paritytech/polkadot-sdk.git", tag = "polkadot-v1.9.0", default-features = false }
 sp-std = { git = "https://github.com/paritytech/polkadot-sdk.git", tag = "polkadot-v1.9.0", default-features = false }
-sp-trie = { git = "https://github.com/paritytech/polkadot-sdk.git", tag = "polkadot-v1.9.0", default-features = false }
+sp-trie = { git = "https://github.com/paritytech/polkadot-sdk.git", version = "29.0.0", default-features = false }
```

Comment:
Added a comment explaining. To clarify this is so that we get the version upgraded we pushed to sp-trie which includes the double ended iterator from trie-db